### PR TITLE
Game jolt | Add Fireside Support

### PR DESCRIPTION
**Pull request by MMDDKK6500**
### Added Fireside Support for the Game jolt presence
ezgif com-gif-maker

Fixed some spacing issues, the "pages" const that had no type defined to it.
Updated the logo and thumbnail, the thumbnail was from a outdated version of Game jolt and the logo was not the correct aspect ratio(1:1), they were also using `premid.is-inside.me` instead of `i.imgur.com`.

### CODE REVIEW

**Review by Slowlife01 — DISMISSED** [review]

File: websites/G/Game Jolt/presence.ts [hunk 1]
```diff
@@ -29,25 +40,48 @@ presence.on("UpdateData", async () => {
...
+      const firesideOwner = document.querySelector(
+          "#content > div > div > div > div > h2 > small > a"
+        ).getAttribute("href").slice(1),
+        fireside = document.querySelector(
+          "#content > div > div > div > div > h2"
+        ).textContent
+          .replace("\n\t\t\t\t\t\t\t", "")
+          .replace("\n\t\t\t\t\t\t","")
+          .replace("\n\t\t\t\t\t", "")
+          .replace("\n\t\t\t\t", "")
```
Comment (ririxidev) [inline websites/G/Game Jolt/presence.ts:69]: umm, please use regex here :)

File: websites/G/Game Jolt/presence.ts [hunk 2]
```diff
@@ -29,25 +40,48 @@ presence.on("UpdateData", async () => {
...
     profile = document.querySelector(
       "#content > div > div > header > section > div > div.row > div > div > h1 > small"
-    ) as HTMLElement;
+    ) as HTMLElement,
 
-  const data: { [k: string]: any } = {
-    largeImageKey: "gj-logo",
-    startTimestamp: Math.floor(Date.now() / 1000)
-  };
+    data: { [k: string]: string | number } = {
```
Comment (ririxidev) [inline websites/G/Game Jolt/presence.ts:45]:
```suggestion
    data: PresenceData = {
```

File: websites/G/Game Jolt/dist/metadata.json [hunk 3]
```diff
@@ -10,9 +10,9 @@
     "nl": "Game Jolt is een hostingdienst voor freeware en commerciële videospelletjes met sociale functies."
   },
   "url": "gamejolt.com",
-  "version": "1.2.7",
-  "logo": "https://premid.is-inside.me/EOMHkNul.png",
-  "thumbnail": "https://premid.is-inside.me/ncJH8KMV.png",
+  "version": "1.3.0",
```
Comment (ririxidev) [inline websites/G/Game Jolt/dist/metadata.json:13]:
```suggestion
  "version": "1.2.8",
```

File: websites/G/Game Jolt/presence.ts [hunk 4]
```diff
@@ -65,14 +99,15 @@ presence.on("UpdateData", async () => {
...
     data.state = profile.textContent;
   } else {
     data.details = "Viewing a page:";
     data.state = "Home";
   }
 
-  if (data.details && data.state && data.details != "" && data.state != "")
+  if (data.details && data.state) 
     presence.setActivity(data);
+
```
Comment (ririxidev) [inline websites/G/Game Jolt/presence.ts:112]: why space here?

**Review by rxri — CHANGES_REQUESTED** [review]

**Review by Bas950 — APPROVED** [review]

**Review by rxri — APPROVED** [review]

**Pull request merged**